### PR TITLE
Adapt install page text to recent move of the install FAQ

### PR DESCRIPTION
**Pull request by eLtMosen**
The main install page now asks to make sure to be familiar with the FAQ before installing. 
Making the individual sentences on each install page redundant.

### CODE REVIEW

File: templates/layouts/aw-install-simg.hbs [hunk 1]
```diff
@@ -12,7 +12,7 @@
     {{!-- Content --}}
     <div class="container docs-container">
       <h1>Hardware Support</h1>
-      <p>Before installing AsteroidOS to your watch, make sure you are aware of the capabilities and limitations of AsteroidOS on the {{title}}. The following table should summarize the current support of this watch:</p>
+      <p>The following table is a summary of the currently supported features of the {{title}}.</p>
```
Comment (beroset) [inline templates/layouts/aw-install-simg.hbs:15]: I think the text is not quite right.  It's not just currently supported features, but also unsupported features that are listed.  Maybe better would be "The following table summarizes the state of the features of {{title}}."

File: templates/layouts/main-install.hbs [hunk 2]
```diff
@@ -15,9 +15,9 @@
 
     {{!-- Content --}}
     <div class="container docs-container">
-      <p>Supported features of all watches are in the <a href="{{rel 'install/features'}}">Features Table</a></p>
-      <p>AsteroidOS can be installed as an alternative operating system on various smartwatches. Instructions for each model can be found below:</p>
-      <p>If you have questions regarding the installation process, please check out the <a href="#FAQ">Install FAQ</a> section.</p>
+      <p>AsteroidOS can be installed as an alternative operating system on various smartwatches. Instructions for each model can be found below.</p>
+      <p>Before installing AsteroidOS, make sure you are aware of the capabilities and limitations of AsteroidOS by consulting the <a href="{{rel 'faq'}}">FAQ page</a>.</p>
+      <p>In case you want to compare supported features of all watches AsteroidOS supports, have a look at the <a href="{{rel 'install/features'}}">Features Table</a>.</p>
```
Comment (beroset) [inline templates/layouts/main-install.hbs:20]: Maybe better would be to be more succinct here.  "A summary of the state of all features of all watches AsteroidOS supports is in the {features table hyperlink}"

**Review by beroset — CHANGES_REQUESTED** [review]
These are just a few small items regarding wording.

File: templates/layouts/aw-install-simg.hbs (hunk 1 above)
Comment (eLtMosen) [reply templates/layouts/aw-install-simg.hbs:15]: Great! That's why i appreciate the review. The sentence also sounded a bit wooden.

**Review by beroset — APPROVED** [review]
Looks good.  Thanks!

**Pull request merged**